packages/react-google-maps-api/src/useLoadScript.md

```diff
@@ -10,7 +10,7 @@ import { GoogleMap, useLoadScript } from '@react-google-maps/api'
 
 function MyComponent() {
   const { isLoaded, loadError } = useLoadScript({
-    googleMapsApiKey="YOUR_API_KEY"
+    googleMapsApiKey: "YOUR_API_KEY"
```

Comment:
Well, it's not all there is wrong, comma missing and there shouldn't be those curly braces on the next line :)